recipes/newrelic/infrastructure/windows.yml

```diff
@@ -78,11 +78,35 @@ install:
     install:
       cmds:
         - powershell -command '(New-Object System.Net.WebClient).DownloadFile("https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.msi", "$env:TEMP\newrelic-infra.msi");'
-        - powershell -command '$LICENSE_KEY="{{.NEW_RELIC_LICENSE_KEY}}"; msiexec.exe /qn /i "$env:TEMP\newrelic-infra.msi" GENERATE_CONFIG=true LICENSE_KEY="$LICENSE_KEY" | Out-Null;'
+        - |
+          powershell -command '
+          $LICENSE_KEY = "{{.NEW_RELIC_LICENSE_KEY}}"
+          $InfraConfig = "C:\\Program Files\\New Relic\\newrelic-infra\\newrelic-infra.yml"
+
+          $NEW_RELIC_REGION = "{{.NEW_RELIC_REGION}}"
+          $STAGING = "false"
+
+          if ($NEW_RELIC_REGION -ilike "staging") {
+            $STAGING = "true"
+          }
+
+          if (Test-Path $InfraConfig) {
+            msiexec.exe /qn /i "$env:TEMP\newrelic-infra.msi" | Out-Null
+
+            (Get-Content $InfraConfig) | Foreach-Object {
+              $_ -replace "^license_key: .*", ("license_key: " + $LICENSE_KEY) `
+                 -replace "^enable_process_metrics: .*", "enable_process_metrics: true" `
+                 -replace "^staging: .*", ("staging: " + $STAGING)
```

Comment:
@noahmmcgivern  the staging element might not be there if this was a production deploy followed by a staging deploy.